securedrop-admin

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python2
+"""
+SecureDrop Admin Toolkit.
+
+For use by administrators to install, maintain, and manage their SD
+instances.
+"""
+
+import argparse
+import subprocess
+import os
+import sys
+import getpass
+
+
+SD_DIR = os.path.dirname(os.path.realpath(__file__))
+ANSIBLE_PATH = os.path.join(SD_DIR, "./install_files/ansible-base")
+SITE_CONFIG = os.path.join(ANSIBLE_PATH, "group_vars/all/site-specific")
+VENV_DIR = os.path.join(SD_DIR, ".venv")
+VENV_ACTIVATION = os.path.join(VENV_DIR, 'bin/activate_this.py')
+
+
+def error(msg, level='warning', exit=True):
```

Comment:
I agree with @fowlslegs that the Python logging module is generally more flexible/extensible in the long run, but think migrating to using the logging module can wait for a later PR.